ko/sql/identifier.rst

```diff
@@ -49,6 +49,20 @@
     "abc""def"
     [position]
 
+큰따옴표, 대괄호, 백틱 부호로 둘러싸인 식별자라도 대괄호([ ])나 점(.)을 포함할 수 없다.
+
+*  대괄호는 내부에서 쿼리를 파싱할 때 예약어 처리를 위해 기본으로 사용한다.
+*  점은 Path Expression 또는 JSON에서 사용되며, 스키마 이름과 객체 이름을 구분하는데에도 사용된다.
+
+다음은 큰따옴표(" ")로 둘러쌓여 있지만 허용되지 않는 식별자이다.
+
+::
+
+    "he[[o"
+    "wor]d"
+    "hello[]world"
+    "hello.world"
+
```

Comment:
How about handling warnign visibly?
`.. Warning::`

Reply:
Thanks, I'll change it.